web/shake.js

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
I'm sure this is impeccable since we didn't write it

Reply:
yes